src/views/Proposal.vue

```diff
@@ -14,7 +14,6 @@
         <template v-if="loaded">
           <h1 class="mb-2">
```

Comment:
You can simplify here and just use this
```suggestion
          <h1 v-text="proposal.title" class="mb-2"/>
```